src/components/GoodsList/GoodsList.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+
+type Props = {
+  goods: string[];
+};
+
+export const GoodsList: React.FC<Props> = ({ goods }) => {
+  const [goodsList, setGoodsList] = useState(goods);
+
+  const reverseList = () => (
+    setGoodsList([...goods].reverse())
```

Comment:
in this implementation your reverse method is a one-use - try it on deployed version
that's because you're referring to `goods` vs referring to goods in your component state - `goodsList`

make `reverse` multi-usable

Reply:
Fixed, thank you.